app/controllers/admin/countries_controller.rb

```diff
@@ -71,4 +75,13 @@ def set_country
     def country_params
       params.require(:country).permit(:name, :iso, :shp_url, :geojson_url)
     end
+
+  private
+    def sort_column
+      Country.column_names.include?(params[:sort]) ? params[:sort] : "name"
+    end
+
+    def sort_direction
```

Comment:
as discussed you could move this method to the AdminController and would be available here and in the other controllers that inherit from it.